frame/bridge/ethereum/backing/src/lib.rs

```diff
@@ -249,9 +290,17 @@ decl_module! {
 				let module_event: <T as Trait>::Event = raw_event.clone().into();
 				let system_event: <T as frame_system::Trait>::Event = module_event.into();
 
+				locked = true;
+
 				<LockAssetEvents<T>>::append(system_event);
 				Self::deposit_event(raw_event);
 			}
+
+			if locked {
+				if let Some(mmr_root) = T::DarwiniaMMR::get_root() {
```

Comment:
OK, here we need to sign for each mmr root, does this mmr root enclose the lock event happened in this block?

mmr root is generated after block execution?

Reply:
```
1. Event.block_height

Only trigger block_to_sign (= Event.block_height + 1) after minimum interval(10 blocks) from last_block_to_sign



2. block_to_sign, Duration = 30 blocks
```